### PR TITLE
Update to new design

### PR DESCRIPTION
**Pull request by psjnek**
This PR updates the design of the FreeOTP website.

Screenshot:
image

### CODE REVIEW

**Comment by psjnek** [comment]
@justin-stephenson 
Could you maybe take a look at this PR?

File: index.html [hunk 1]
```diff
@@ -1,95 +1,115 @@
...
+          </div>
+          <div class="bg-body-tertiary col rounded m-2 p-4" id="encryption">
+            <div class="mb-4 text-center"><i class="bi bi-qr-code-scan fs-1"></i></div>
+            <h3>Simple Setup</h3>
+            <p class="text-secondary">FreeOTP supports scaning a QR code to setup new tokens.</p>
+          </div>
+          <div class="bg-body-tertiary col rounded m-2 p-4" id="encryption">
+            <div class="mb-4 text-center"><i class="bi bi-fingerprint fs-1"></i></div>
+            <h3>Encryption</h3>
+            <p class="text-secondary">All of your tokens are encrypted by default. Just select a password that you would
```
Comment (justin-stephenson) [inline index.html]: I would change this text to state that Tokens are stored in the device Keystore, enhancing security by making them more difficult to extract from the device.

**Comment by justin-stephenson** [comment]
@WebSnke Thank you for the PR, can we add links to the Google Play and iOS App store ? What will be the `download.html` file that is currently empty?

image

I see what looks like a border/margin issues here (I am using firefox). 

Please also link to the Privacy Policy https://freeotp.github.io/privacy.html

Otherwise it looks great to me.

**Comment by psjnek** [comment]
> @WebSnke Thank you for the PR, can we add links to the Google Play and iOS App store ? What will be the `download.html` file that is currently empty?
> 
> image
> 
> I see what looks like a border/margin issues here (I am using firefox).
> 
> Please also link to the Privacy Policy [freeotp.github.io/privacy.html](https://freeotp.github.io/privacy.html)
> 
> Otherwise it looks great to me.

I already linked to the privacy policy in the footer. What do you mean by "border/margin issues"?

**Comment by psjnek** [comment]
> @WebSnke Thank you for the PR, can we add links to the Google Play and iOS App store ? What will be the `download.html` file that is currently empty?
> 
> image
> 
> I see what looks like a border/margin issues here (I am using firefox).
> 
> Please also link to the Privacy Policy [freeotp.github.io/privacy.html](https://freeotp.github.io/privacy.html)
> 
> Otherwise it looks great to me.

The download.html is supposed to contain the links for downloading FreeOTP. Do you think I should just add the banners on the index.html instead?

Update: I just tried to create a embeded banner for the Google Play Store and the App Store but it turns out that the Google Play Store uses a default padding in the image itself. This sucks because the banner would have different heights.

**Comment by psjnek** [comment]
I would like to start with a new PR for updating the design as I think that this commit history got a bit messy...

**Pull request closed without merging**